Debian-stable-cross-compilation-for-arm/startup.sh

```diff
@@ -8,8 +8,7 @@ cp /tmp_ssh/* ~/.ssh
 cd /cgal_root/
 source "env.sh"
 #launch script
-bash $CGAL_DIR/developer_scripts/run_testsuite_with_cmake
+bash $CGAL_DIR/${SCRIPTS_DIR}developer_scripts/run_testsuite_with_cmake
```

Comment:
Are you sure? Should not it be:
```suggestion
bash $CGAL_DIR/${SCRIPTS_DIR}/run_testsuite_with_cmake
```
or 
```suggestion
bash ${SCRIPTS_DIR}/run_testsuite_with_cmake
```
?

Reply:
to avoid a `//` in the path when SCRIPTS_DIR does not exist, I define it as `Scripts/` in the variable. i know that `path//to/file` is the same than 'path/to/file' but I find it ugly.